101-maps-create/azuredeploy.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "account_name":{
+            "type": "string",
+            "defaultValue": "exampleMap",
```

Comment:
does this need to be globally unique?  If so, use uniqueString and update the description